### ISSUE TITLE
Release 5.2.0 does not have color-adjust deprecation fix

### ISSUE THREAD

**Issue by Vheissu** [twbs/bootstrap#36797]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

As you can see, a PR was merged which removes the `color-adjust` property inside of the `_form-check.scss` file for the rule `.form-check-input` [here](https://github.com/twbs/bootstrap/pull/36258).

However, I import the compiled CSS files in my projects because I am using Autoprefixer and PostCSS.  I import CSS like this:

```
import 'bootstrap/dist/css/bootstrap.css';
```

As you can see in the compiled CSS file, `color-adjust` is still there: https://github.com/twbs/bootstrap/blob/main/dist/css/bootstrap.css#L2274

Furthermore, looking in the `SCSS` directory inside of my `node_modules` directory is showing that the `color-adjust` property has been removed from the `scss` file also.  But, it's in the compiled CSS file.

So, either something has gone astray during the build process, or a tool is adding `color-adjust`. As a result, in my application that uses Autoprefixer 10.4.7, I am getting the warning: `autoprefixer: Replace color-adjust to print-color-adjust. The color-adjust shorthand is currently deprecated.` which is tripping up my continuous integration builds as warnings are considered errors in production release environments.

I looked through the releases for 5.2 beta and 5.2 and could see no fixes for this issue mentioned in the notes.

### Reduced test cases

Numerous cases have already been outlined in previously closed issues:

- https://github.com/twbs/bootstrap/issues/36259

### What operating system(s) are you seeing the problem on?

Windows, macOS, Linux

### What version of Bootstrap are you using?

5.2.0

**Comment by julien-deramond** [comment]
We might have the same issue in v4:

https://github.com/twbs/bootstrap/blob/e5643aaa89eb67327a5b4abe7db976f0ea276b70/dist/css/bootstrap.css#L3798

**Comment by julien-deramond** [comment]
Some extra-information about this issue. Even with Autoprefixer 10.4.7, apparently there are some cases where `color-adjust: exact;` is added.

It can be tested directly in https://autoprefixer.github.io/.

As an example:
```css
.example {
    print-color-adjust: exact;
}
```

* With the basic rule proposed by the website for `browserlist`, `color-adjust: exact` is not added.
* With the content of our `.browserlistrc` (`last 2 major versions, not dead, Chrome >= 60, Firefox >= 60, Firefox ESR, iOS >= 12, Safari >= 12, not Explorer <= 11`), `color-adjust: exact` is added.
* If we remove the rules regarding Firefox from our `.browserlistrc` (`last 2 major versions, not dead, Chrome >= 60, iOS >= 12, Safari >= 12, not Explorer <= 11`), `color-adjust: exact` is not added.

**Comment by Vheissu** [comment]
@julien-deramond Some great investigative work there. Some very confusing contributing factors here that make it hard to know what the final fix is. But, good to see the cause of the property has been discovered.

**Comment by julien-deramond** [comment]
FYI I've created https://github.com/postcss/autoprefixer/issues/1471 where I tried to gather the elements of my analysis

**Comment by julien-deramond** [comment]
Good news, it's been fixed in `autoprefixer` 10.4.8. Dependency version will be updated in Bootstrap. In your projects, you probably can force it to use the 10.4.8 so that the warning message disappears.

**Comment by Vheissu** [comment]
@julien-deramond Perfect. I confirm the issue is fixed in Autoprefixer 10.4.8.
